src/main/java/no/acat/configuration/AppProperties.java

```diff
@@ -0,0 +1,18 @@
+package no.acat.configuration;
+
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+@ConfigurationProperties(prefix = "application")
+public class AppProperties {
+    private String apiRegistrationsFile;
+
+    public String getApiRegistrationsFile() {
+        return apiRegistrationsFile;
+    }
+
+    public void setApiRegistrationsFile(String apiRegistrationsFile) {
+        this.apiRegistrationsFile = apiRegistrationsFile;
+    }
```

Comment:
Vurder å erstatte disse med lombok Getter and Setter

Reply:
erstattet